nncf/openvino/graph/node_utils.py

```diff
@@ -337,3 +337,47 @@ def get_weight_channel_axes(node: NNCFNode, weights_port_id: int) -> List[int]:
             channel_axes.append(matmul_channel_axis)
 
     return channel_axes
+
+
+def get_activation_channel_axis(node: NNCFNode) -> int:
+    """
+    Returns axis number of the activation tensor which correspond to it channel.
+
+    :param node: NNCFNode instance.
+    :return: Channel axis number.
+    """
+    channel_axis = node.metatype.input_channel_axis if node.metatype.input_channel_axis is not None else 1
```

Comment:
Why don't we reuse `get_output_channel_axis` here?

Reply:
Because `get_activation_channel_axis` need's `activation` aka input channel axis. I'll rename this method.